integration/spark-common/src/main/scala/org/apache/spark/sql/catalyst/CarbonDDLSqlParser.scala

```diff
@@ -787,6 +787,19 @@ abstract class CarbonDDLSqlParser extends AbstractCarbonSparkSQLParser {
       }
     }
 
+    // range_column should be there in create table cols
+    if (tableProperties.get(CarbonCommonConstants.RANGE_COLUMN).isDefined) {
+      val rangeColumn = tableProperties.get(CarbonCommonConstants.RANGE_COLUMN).get.trim
```

Comment:
I think we can also check the number of fields and complain that carbondata only support specifying one fields now.

Reply:
ok